tests/system_tests/test_plans_system.py

```diff
@@ -28,30 +28,96 @@ def test_spectroscopy(
     events = bluesky_plan_runner.run(
         TaskRequest(
             name="spectroscopy",
-            params={},
             instrument_session=latest_commissioning_instrument_session,
         ),
         timeout=10,
     )
     assert events["FINISHED"][0]["scanDimensions"] == [5]
 
 
+def test_spectroscopy_with_custom_trajectory(
+    bluesky_plan_runner: BlueskyPlanRunner, latest_commissioning_instrument_session: str
+):
+    scan_spec = Line("sample_stage.x", 0.0, 5.0, 5) * Line(
+        "sample_stage.y", 2.0, 5.0, 3
+    )
```

Comment:
Does this make `x` the outer dimension and is that what we want? Does it matter?

Reply:
Yes and I can't think of any immediate reason why it would